src/conn/conn_api.c

```diff
@@ -2128,6 +2157,31 @@ __debug_mode_log_retention_config(WT_SESSION_IMPL *session, const char *cfg[])
     return (ret);
 }
 
+/*
+ * __debug_mode_api_call_count_config --
+ *     Set the debug configurations for api call counts.
+ */
+static int
+__debug_mode_api_call_count_config(WT_SESSION_IMPL *session, const char *cfg[])
+{
+    WT_CONFIG_ITEM cval;
+    WT_CONNECTION_IMPL *conn;
+
+    conn = S2C(session);
+
+    WT_RET(__wt_config_gets(session, cfg, "debug_mode.api_call_count", &cval));
+    if (cval.val) {
+        FLD_SET(conn->debug_flags, WT_CONN_DEBUG_API_ENTRY_COUNT);
+        /* Allocate the API call count tracker. */
+        WT_RET(__wt_calloc_one(session, &conn->api_call_tracker));
+    } else if (!FLD_ISSET(conn->debug_flags, WT_CONN_DEBUG_API_ENTRY_COUNT)) {
```

Comment:
Change !FLD_ISSET(conn->debug_flags, WT_CONN_DEBUG_API_ENTRY_COUNT)
to FLD_ISSET(conn->debug_flags, WT_CONN_DEBUG_API_ENTRY_COUNT)